src/vaults/vaults.utils.ts

```diff
@@ -127,6 +133,19 @@ export function getVaultState(status: number): VaultState {
   }
 }
 
+/**
+ * Convert on chain state to vault state enumeration for v2 registry
+ * @param status On chain state
+ * @returns Vault state enumeration
+ */
+export function getVaultRegv2State(status: number): VaultState {
+  return vaultToChainEnumStateList[status] || VaultState.Deprecated;
+}
+
+export function apiVaultStatusToChainValue(status: VaultState): number {
+  return vaultToChainEnumStateList.indexOf(status) || VaultStatus.deprecated;
```

Comment:
same here, we using discontinued now on the api only 
https://github.com/Badger-Finance/badger-sdk/blob/main/src/api/enums/vault-state.enum.ts